contracts/transfer/src/wasm/bridge.rs

```diff
@@ -4,77 +4,25 @@
 //
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
-use crate::{ops, TransferContract};
+use crate::{Call, TransferContract};
 
 use canonical::{
     BridgeStore as BridgeStoreCanon, ByteSink, ByteSource, Canon, Id32, Store,
 };
 use dusk_abi::{ContractState, ReturnValue};
 
-const PAGE_SIZE: usize = 1024 * 8;
+const PAGE_SIZE: usize = 1024 * 32;
 
 type BridgeStore = BridgeStoreCanon<Id32>;
 
 #[no_mangle]
-fn q(bytes: &mut [u8; PAGE_SIZE]) {
-    if let Err(_) = query(bytes) {
-        // TODO handle error
-        // https://github.com/dusk-network/rusk/issues/193
-    }
+fn q(_bytes: &mut [u8; PAGE_SIZE]) {
+    panic!("Spurious functions are supposed to be called from the state of the contract!");
```

Comment:
Bear in mind that this might not be exactly correct. Since queries are not exactly the same (in purpose as spurious functions). 

Anyway, if this contract supports no queries, we should basically remove this fn from the contract directly.

Reply:
I believe the function is mandatory because its part of the API